### PR TITLE
updated the external_auth_ipa test.

### PR DESCRIPTION
**Pull request by amavinag**
* the hostname is not correctly updated through
    appliance_console, workaround is been added to the test.
  * FQDN is a must for extrenal, which is included in the test.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 97d0706410385e694c6159d8f3a4771f1256f52c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 027823b3b9de49c7cea336c1c029cd1267ca82b8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1144fd31baf53f4c6532a27f30253a605e1ec4f0:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/ext_auth.py [hunk 1]
```diff
@@ -18,7 +19,25 @@ def setup_external_auth_ipa(**data):
...
+    connect_kwargs = {
+        'username': credentials['host_default']['username'],
+        'password': credentials['host_default']['password'],
+        'hostname': data['ipaserver'],
+    }
+    import fauxfactory
+    appliance_name = 'cfmeappliance' + (fauxfactory.gen_alpha(7)).lower()
+    appliance_address = appliance.IPAppliance().address
+    appliance_fqdn = appliance_name + '.' + data['iparealm'].lower()
+    ssh = SSHClient(**connect_kwargs)
```
Comment (psav) [inline utils/ext_auth.py]: We should not be instantiating an ssh client here, should we?
In actual fact, this function should be inside utils.appliance and the script should call out to it like in all the other instances, care to do that while you're here to help technical debt?

Comment (amavinag) [inline utils/ext_auth.py]: @psav, line#31 is ssh for ipaserver and not for appliance. Let me know if we still need to move this to appliance.py (this method functionality is not related to appliance).

Comment (psav) [inline utils/ext_auth.py]: Ahhh that's what I thought - maybe make this clearer by the name of the variable

**Comment by dajoRH** [comment]
Lint report for commit d91d9fc553f0872ada92460b269bb67384d9f33c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1c76e56982b7d4cab1da46d7788a397381985512:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e02cd329669aad738a8152c36b180c8af88d1976:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/ext_auth.py [hunk 2]
```diff
@@ -18,7 +19,25 @@ def setup_external_auth_ipa(**data):
         iparealm: Realm.
         credentials: Key of the credential in credentials.yaml
     """
+    connect_kwargs = {
+        'username': credentials['host_default']['username'],
+        'password': credentials['host_default']['password'],
+        'hostname': data['ipaserver'],
+    }
+    import fauxfactory
+    appliance_name = 'cfmeappliance' + (fauxfactory.gen_alpha(7)).lower()
```
Comment (mfalesni) [inline utils/ext_auth.py]:
```python
appliance_name = 'cfmeappliance'.format(fauxfactory.gen_alpha(7).lower())
```

File: utils/ext_auth.py [hunk 3]
```diff
@@ -18,7 +19,25 @@ def setup_external_auth_ipa(**data):
...
+    appliance_address = appliance.IPAppliance().address
+    appliance_fqdn = appliance_name + '.' + data['iparealm'].lower()
```
Comment (mfalesni) [inline utils/ext_auth.py]:
```python
appliance_fqdn = '{}.{}'.format(appliance_name, data['iparealm'].lower())
```

**Comment by dajoRH** [comment]
Lint report for commit 2075fbfd275136209902eb1126c249e962543bf4:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b9ba94ff9a92d562d0a56712ab3c151774c7884b:

Everything seems all right :smile: .
*CFME QE Bot*
